package.json

```diff
@@ -683,7 +713,7 @@
       "view/item/context": [
         {
           "command": "confluent.copyResourceId",
-          "when": "view in confluent.viewsWithResources && viewItem in confluent.resourcesWithIDs",
+          "when": "view in confluent.viewsWithResources && (viewItem in confluent.resourcesWithIDs || viewItem =~ /.*-schema$/)",
```

Comment:
Schema context values have gotten more complex and are no longer expressible within `resourcesWithIDs`.